Lib/shutil.py

```diff
@@ -73,8 +73,13 @@ class RegistryError(Exception):
     and unpacking registries fails"""
 
 
-def copyfileobj(fsrc, fdst, length=16*1024):
-    """copy data from file-like object fsrc to file-like object fdst"""
+def copyfileobj(fsrc, fdst, length=None):
+    """Copy data from file-like object `fsrc` to file-like object `fdst`.
+
+    An in-memory buffer size can be set with `length`; the default is 16 KB.
```

Comment:
please mention the unit, "size in bytes"

Reply:
I encourage you to use _KiB_, which other parts of the documentation already use. If you write _16 KB_ it could easily be interpreted as 16,000 B (lower-case _k_ means 1000, as in _km_ for kilometre).